pypeit/images/combineimage.py

```diff
@@ -46,24 +39,25 @@ class CombineImage:
             The 1-indexed detector number(s) to process.
         par (:class:`~pypeit.par.pypeitpar.ProcessImagesPar`):
             Parameters that dictate the processing of the images.
-        files (:obj:`list`):
-            A set of one or more images to process and combine.
+        rawImages (:obj:`list`):
+            A list of one or more :class:`~pypeit.images.rawimage.RawImage` objects 
+            to be combined.             
 
     """
-    def __init__(self, spectrograph, det, par, files):
-        self.spectrograph = spectrograph
-        self.det = det
+    def __init__(self, rawImages, spectrograph, par):
```

Comment:
Importantly, `spectrograph` is *not* an attribute of each `PypeItImage` (i.e. the actual type of every element of `rawImages`). But, if we move the lamp check out of this class, we only need the spectrograph name, which we can get from `PypeItImage.PYP_SPEC`.  If we do that, the instantiation arguments are only `rawImages` and `par`.

Reply:
Great suggestion. This was also my hope, i.e. to remove the dependency on spectrograph initially. Now that the lamp check is out, I've removed it as a dependency.